src/System.Text.RegularExpressions/tests/RegexGroupCollectionTests.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.Collections.Generic;
+using System.Text.RegularExpressions;
+using Xunit;
+
+public class RegexGroupCollectionTests
+{
+    [Fact]
+    public static void GetGroupTest()
+    {
+        var caps = new Dictionary<int, int>
+        {
+            { 0, 0 }
+        };
+        var groups = new GroupCollection(null, caps);
```

Comment:
Yes, that was on my plans, but I can't even seem to run this simple test.
Was hoping I could see why on appveyor or jenkins, but it looks like it's disabled?